From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sevenrock <sevenrock@hotmail.de>
Date: Wed, 1 Jan 2025 21:09:27 +0100
Subject: [PATCH] prepare techpack merge: Revert "camera: ois: avoid ois power down twice when download
 firmware failed"

This reverts commit dd8ca877e397393df9a43978509771bbf9d7abf5.

Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 drivers/cam_sensor_module/cam_ois/cam_ois_core.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/cam_sensor_module/cam_ois/cam_ois_core.c b/drivers/cam_sensor_module/cam_ois/cam_ois_core.c
index 189ce58fefb3..65875a21ca99 100644
--- a/drivers/cam_sensor_module/cam_ois/cam_ois_core.c
+++ b/drivers/cam_sensor_module/cam_ois/cam_ois_core.c
@@ -185,8 +185,6 @@ static int cam_ois_power_up(struct cam_ois_ctrl_t *o_ctrl)
 		return rc;
 	}
 
-	CAM_INFO(CAM_OIS, "OIS Power up successfully");
-
 	rc = camera_io_init(&o_ctrl->io_master_info);
 	if (rc) {
 		CAM_ERR(CAM_OIS, "cci_init failed: rc: %d", rc);
@@ -236,8 +234,6 @@ static int cam_ois_power_down(struct cam_ois_ctrl_t *o_ctrl)
 		return rc;
 	}
 
-	CAM_INFO(CAM_OIS, "OIS power down successed");
-
 #ifdef CONFIG_MOT_DONGWOON_OIS_AF_DRIFT
 	if (strstr(o_ctrl->ois_name, "dw9784"))
 		cam_ois_set_init_info(0);
@@ -1747,7 +1743,7 @@ static int cam_ois_pkt_parse(struct cam_ois_ctrl_t *o_ctrl, void *arg)
 	if (!rc)
 		return rc;
 pwr_dwn:
-	//cam_ois_power_down(o_ctrl); /* ois will pown down in CAM_RELEASE_DEV when closed camera */
+	cam_ois_power_down(o_ctrl);
 	return rc;
 }
 
-- 
2.39.5

